cfme/services/catalogs/ansible_catalog_item.py

```diff
@@ -28,6 +28,81 @@ class BootstrapSelect(VanillaBootstrapSelect):
     ROOT = ParametrizedLocator('.//select[normalize-space(@name)={@id|quote}]/..')
 
 
+class ActionsCell(View):
+    edit = Button(
+        **{"ng-click": ParametrizedString(
```

Comment:
@mfalesni this patter looks painful - can we provide something to implement this better

passing kwargs along that aren't valid identifies seems just missplaced, but currently there seems no other possiblity

Reply:
Button could `s/_/-/` on kwargs to be converted to attribute matchers since underscores are not used in html attributes.

Reply:
I have a PR open for that in widgetastic_patternfly.